frontend/templates/product/sections/ProductOverviewSection/CompatibilityNotes.tsx

```diff
@@ -0,0 +1,41 @@
+import {
+   AccordionButton,
+   AccordionIcon,
+   AccordionItem,
+   AccordionPanel,
+   Box,
+} from '@chakra-ui/react';
+import type { Product } from '@pages/api/nextjs/cache/product';
+import React from 'react';
+import { SplitCompatibilityNotes } from '../CompatibilityNotesSection';
+
+export type CompatibilityNotesProps = {
+   product: Product;
+};
+
+export function CompatibilityNotes({ product }: CompatibilityNotesProps) {
+   const compatibilityNotes = product.compatibilityNotes;
+   const [visibleDevices, hiddenDevices] = SplitCompatibilityNotes({
+      compatibilityNotes,
+   });
+   return (
+      <Box px={2}>
+         <Box pb={5}>{visibleDevices}</Box>
+         <AccordionItem hidden={!hiddenDevices.length}>
+            <AccordionButton py="5" px="1.5">
+               <Box
+                  flex="1"
+                  textAlign="left"
+                  color="gray.800"
+                  fontWeight="semibold"
+                  fontSize="sm"
+               >
+                  {'Show ' + hiddenDevices.length + ' More Models'}
+               </Box>
+               <AccordionIcon />
+            </AccordionButton>
+            <AccordionPanel p={2}>{hiddenDevices}</AccordionPanel>
```

Comment:
How come this one is AccordionPanel and the other one is details + summary?

Reply:
so I like that this section expands smoothly like the rest of the accordion panels in the sidebar. Plus the details + summary will open (on chrome) if somone ctrl f's so we dont need this section to do that